### PR TITLE
Upgrade patches from Chromium 75.0.3770.38 to Chromium 75.0.3770.66

### PR DESCRIPTION
**Pull request by mkarolin**
Fixes brave/brave-browser#4547
Related brave/brave-browser#4651

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [x] Windows
  - [x] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on
  - [x] Windows
  - [x] macOS
  - [x] Linux
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by mkarolin** [comment]
CI build 17 succeeded, the later builds failed for unrelated reasons.

@mihaiplesa, please, take a look at the changes to the mac signing scripts. They are encapsulated in this commit: https://github.com/brave/brave-core/pull/2545/commits/89a64ca407813c7eee4eddb20a996e85b243a4bb

**Comment by mihaiplesa** [comment]
> CI build 17 succeeded, the later builds failed for unrelated reasons.
> 
> @mihaiplesa, please, take a look at the changes to the mac signing scripts. They are encapsulated in this commit: [89a64ca](https://github.com/brave/brave-core/commit/89a64ca407813c7eee4eddb20a996e85b243a4bb)

@simonhong or @emerick might know better.

From what I see the order or args is changing so we'd have to call it differently for `nightly` from Jenkins.

**Comment by mkarolin** [comment]
@mihaiplesa, does the CI call `sign_app.sh` directly? If it just runs `npm run create_dist` then it should be ok.

File: patches/chrome-installer-mac-signing-pipeline.py.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
+index ac3cd9f6b28570410730f8aaaf98b3f7ce468320..6d49ec610df9b9550e1eb2d6febd431eb6e2a724 100644
+--- a/chrome/installer/mac/signing/pipeline.py
++++ b/chrome/installer/mac/signing/pipeline.py
+@@ -85,7 +85,6 @@ def _package_installer_tools(paths, config):
+         'dirdiffer.sh',
+         'dirpatcher.sh',
+         'dmgdiffer.sh',
+-        'keystone_install.sh',
```
Comment (mkarolin) [inline patches/chrome-installer-mac-signing-pipeline.py.patch]: The scripts includes `keystone_install.sh` as an additional tool here, but in `chrome/installer/mac/BUILD.gn` the file is only copied when chrome-branded:
`if (is_chrome_branded) {
    sources += [ "keystone_install.sh" ]
  }`

**Comment by mkarolin** [comment]
Closing in favor of #2585

**Pull request closed without merging**